packages/monaco-editor-react/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to npm module [@typefox/monaco-editor-react](https://www.npmjs.com/package/@typefox/monaco-editor-react) are documented in this file.
 
+## [2.3.0] - 2023-10-xy
+
+- Properly separate and define classic and extended editor [#54](https://github.com/TypeFox/monaco-components/pull/54)
```

Comment:
Note about `vscodeApi` change to `extended` here?

Reply:
I will adjust it accordingly in all CHANGELOGs.

Reply:
Done ⬇️